<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>7</y>
      <width>1331</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="main_layout">
     <item>
      <layout class="QVBoxLayout" name="control_layout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tama√±o:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Porcentaje de hormigas:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_start">
         <property name="text">
          <string>Iniciar/Reiniciar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_pause">
         <property name="text">
          <string>Pausa/Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_color">
         <property name="text">
          <string>Color de la hormiga</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="game_layout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1166</width>
            <height>693</height>
           </rect>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1021</width>
             <height>691</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="gameLayout"/>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
